app/serializers/concerns/user_sidebar_tags_mixin.rb

```diff
@@ -2,10 +2,20 @@
 
 module UserSidebarTagsMixin
   def self.included(base)
+    base.attributes :display_sidebar_tags_section
+
     base.has_many :sidebar_tags, serializer: Sidebar::TagSerializer, embed: :objects
   end
 
   def include_sidebar_tags?
     SiteSetting.enable_experimental_sidebar_hamburger && SiteSetting.tagging_enabled
   end
+
+  def display_sidebar_tags_section
+    DiscourseTagging.filter_visible(Tag, scope).exists?
+  end
+
+  def include_display_sidebar_tags_section?
```

Comment:
This is the same as `include_sidebar_tags?` above, right? I would personally go with the initial name, i.e. I would drop the `_section` suffix here.